### DISCUSSION TITLE
Allow specific users to bypass push limit

### DISCUSSION THREAD

**Discussion by rudyocalan** [community/community#53295] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hey team,

It would be great if some specific people could bypass the limit on how many branches/tags can be updated in a single push. It would help us with our branch cleanup automation.

Thanks!

**Comment by rmdhfz** [comment]
Hi @rudyocalan ,

Unfortunately, there is currently no option to bypass the limit on how many branches/tags can be updated in a single push, even for specific users. This limit is put in place to prevent accidental or malicious modifications to a large number of branches or tags at once. However, you may consider implementing your branch cleanup automation in a way that works within the limits set by the platform, such as breaking the cleanup process into smaller batches or using other tools to accomplish your goal.

**Reply by rudyocalan** [reply]
Thanks @rmdhfz 
Hopefully GitHub team can add that to their roadmap
